README.md

```diff
@@ -37,6 +37,7 @@ The `NewS2DBArrowReader` function takes `S2DBArrowReaderConfig` as a parameter.
 | Args               | nil (no arguments)    | Arguments for placeholder parameters in the query.
 | RecordSize         | 10000                 | The maximum number of rows in the resulting records.
 | ParallelReadConfig | nil (sequential read) | Additional configurations for parallel read. If this value is non-`nil`, parallel read is enabled.
+| UseClientConvesion | false                 | Indicates if the data should be converted to Arrow Record format on the client. It can be set to `true` for test purposes, performance optimizations take place only when it is `false`
```

Comment:
It can also be set in order to use parallel read. I suppose that for big clusters parallel read may give better performance.

Reply:
yes, but parallel read in the dev build we're going to share doesn't seem to be in the best shape, so the main goal now is to use Arrow conversion on server extensively to find bugs, limitations, performance.